adoc/Private-Registry-Administration.adoc

```diff
@@ -743,3 +743,206 @@ suse-registry-harbor-registry-65854df7bc-mrfnj     2/2     Running   0
 suse-registry-harbor-trivy-0                       1/1     Running   1          24h
 suse-registry-harbor-trivy-1                       1/1     Running   0          24h
 ----
+
+=== Backup and Restore
+
+As an application that is deployed on kubernetes, backup and restore for {spr} can be performed by Velero. Velero is an application that performs backup/restore, disaster recovery, and migrate Kubernetes cluster resources and persistent volumes.
```

Comment:
Do we want to link Velero page when it is mentioned for the first time?

Reply:
done